.github/workflows/update-interfaces.yml

```diff
@@ -0,0 +1,42 @@
+name: Update protobuf stubs
+
+on:
+  push:
+  workflow_dispatch:
+
+jobs:
+  update_proto:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: recursive
+    - name: Copy files
+      run: |
+        SRC=./api-interfaces/gooseai
+        TGT=./src/stability_sdk/interfaces/gooseai
+        KIND=generation
+        cp $SRC/$KIND/${KIND}_pb2_grpc.py $TGT/$KIND/${KIND}_pb2_grpc.py
+        cp $SRC/$KIND/${KIND}_pb2.py $TGT/$KIND/${KIND}_pb2.py
+        #############################
+        # for backwards compatibility
```

Comment:
I'm inclined to agree.  Since people can now pull these generated files out of the `api-interfaces`, I think it makes sense to only have this project be concerned about the generated files meant for python consumption.

Like Chia said, we'll want to cut a major version so we don't break existing consumers.

Reply:
it's been less than a month since the beta release so the timing for a major version is good too. especially considering the third party ts client is currently broken because of this dependency already. maybe we do this PR leaving the backwards compatibility stuff in it as a temporary fix to get people who are using the third party ts client back on their feet, and then follow that up with a new major version. this would give a little bit of breathing room to users depending on the current structure to upgrade without requiring that their stuff be broken and unavilable while they're in transition

Reply:
alternatively, choosing not to support backwards compatibility is a good forcing function for upgrading